Zend/zend_compile.c

```diff
@@ -8530,6 +8530,27 @@ void zend_compile_greater(znode *result, zend_ast *ast) /* {{{ */
 }
 /* }}} */
 
+/* We do not use zend_compile_binary_op for this because we want to retain the left-to-right
+ * evaluation order. */
+void zend_compile_pipe(znode *result, zend_ast *ast) /* {{{ */
+{
+	zend_ast *expr_ast = ast->child[0];
+	zend_ast *func_ast = ast->child[1];
+
+	znode expr_node;
+
+	zend_compile_expr(&expr_node, expr_ast);
+
+	zend_ast *call =  zend_ast_create(ZEND_AST_CALL,
+		func_ast,
+		zend_ast_create_list(1,
+		ZEND_AST_ARG_LIST,
```

Comment:
Is this indentation correct...? Looks weird, but I don't know which style is used in php-src.

Reply:
It's a single tab each step, which is what that file uses, because whatever reason.